test/adapter/json/polymorphic_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class PolymorphicTest < ActiveSupport::TestCase
+          def setup
```

Comment:
@marcgarreau Nothing wrong with the way you did it. It just happens that you could also use the block syntax as @bf4 suggested in order to avoid an explicit call to `super`.